### PR TITLE
Make it obvious which round the proposal will be submitted

### LINKED ISSUE #60: Submit Proposal UX - Update the UX flow so users cannot submit a proposal unless previous proposals have been completed
### Problem:
We are planning to make it so participants can submit to the next funding round, as soon as submissions close. In order to make this obvious to users, communicate visually to users which round their proposals will be submitted into, in addition to other information that might be useful such as deadline.

### DoD:
- [ ] When I create a new proposal, the portal makes it obvious which funding round my proposal will be submitted to

### PR DESCRIPTION
**Pull request by trizin**
Closes #60 

+ Redis client & service
+ `/round` endpoint
+ Make it obvious which round the proposal will be submitted.

image
image

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
It's looking good for resolving this issue, but looking at the bigger picture I'll suggest fetching the entire Funding Round object(in the home page maybe) and store it. We would need it in the future improvements when we want to show more information about the Funding Round, and then just an api for getting the number won't be necessary.

File: backend/utils/redis/round.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const { getCurrentRound } = require("../airtable/utils");
+const redis = require("./index");
+
+async function getCurrentRoundNumber() {
```
Comment (idiom-bytes) [inline backend/utils/redis/round.js:4]: Awesome!

File: docker-compose.yml [hunk 2]
```diff
@@ -13,5 +13,14 @@ services:
...
     volumes:
       - mongodb:/data/db
+
+  redis:
+    image: redis:alpine
+    ports:
+      - 6379:6379
+    command: redis-server --save 20 1 --loglevel warning
+    volumes:
+      - cache:/data
```
Comment (idiom-bytes) [inline docker-compose.yml:23]: Awesome work. This will be great, looking forward to expanding the cache!

**Comment by idiom-bytes** [comment]
Awesome update. Great add to the docker/yaml + BE.

**Review by idiom-bytes — APPROVED** [review]
Looks great!

**Pull request merged**